banyan-cli/src/main.rs

```diff
@@ -184,7 +201,7 @@ impl Tagger {
     }
 }
 
-fn create_salsa_key(text: &str) -> salsa20::Key {
+fn create_salsa_key(text: String) -> salsa20::Key {
```

Comment:
Why make this a String? The method does not seem to need to own the thing.

Reply:
Because structopt yields a String :man_shrugging: